### PR TITLE
Update readme fixing #13

### PR DESCRIPTION
**Pull request by sanchit-bansal**

### CODE REVIEW

**Comment by butlerx** [comment]
could you add updated screenshots

**Comment by sanchit-bansal** [comment]
Sure, can you guide me to a url of this installation or help me install the same ?

**Comment by butlerx** [comment]
sure just run `npm install` and `npm start` from this https://github.com/redbrick/static-site

**Comment by butlerx** [comment]
its actually in production on https://alpha.redbrick.dcu.ie could you also update the preview link

File: README.md [hunk 1]
```diff
@@ -1,9 +1,10 @@
 #Icarus
 
-### The blog theme you may fall in love with, coming to Hexo. [Preview](http://ppoffice.github.io/hexo-theme-icarus/)
-![](http://ppoffice.github.io/hexo-theme-icarus/gallery/preview.png "")
+### The blog theme you may fall in love with, coming to Hexo. 
+### Click here to [Preview](https://alpha.redbrick.dcu.ie/) the theme.
+![](http://image.prntscr.com/image/4db6552e12434ae9b405bb301f864a1b.png "")
 
-#### [View Documentation](https://github.com/ppoffice/hexo-theme-icarus/wiki)
+#### [View Documentation](hhttps://github.com/redbrick/hexo-theme-icarus/wiki)
```
Comment (butlerx) [inline README.md]: https :)

Comment (sanchit-bansal) [reply README.md]: Sure :)

**Comment by sanchit-bansal** [comment]
For some reason, I can't find any gallery screenshots sorry :(

**Comment by butlerx** [comment]
That's fine that's not changed

**Review by butlerx — APPROVED** [review]

**Pull request merged**